openapi.json

```diff
@@ -675,6 +675,100 @@
         ]
       }
     },
+    "/opening_hours/instances": {
+      "get": {
+        "parameters": [
+          {
+            "name": "_format",
+            "in": "query",
+            "type": "string",
+            "enum": ["json"],
+            "required": true,
+            "description": "Request format",
+            "default": "json"
+          }, 
+          {
+            "name": "from_date",
+            "type": "string",
+            "format": "date",
+            "description": "Retrieve opening hours which occur after and including the provided date. In ISO 8601 format.",
+            "in": "query",
+            "required": false
+          }, 
+          {
+            "name": "to_date",
+            "type": "string",
+            "format": "date",
+            "description": "Retrieve opening hours which occur before and including the provided date. In ISO 8601 format.",
+            "in": "query",
+            "required": false
```

Comment:
I am not sure this changed between reviews but I notice that the two parameters are not mentioned as required but we throw an exception if they are not provided. That does not seem right.

Reply:
Whoops, good catch. my bad